include/tfm/tfm_ioctl_api.h

```diff
@@ -48,6 +49,31 @@ struct tfm_read_service_out_t {
 	uint32_t result;
 };
 
+enum tfm_gpio_service_type {
+	/** Select which MCU / Subsystem controls the pin */
+	TFM_GPIO_SERVICE_TYPE_PIN_MCU_SELECT = 0,
+};
+
+/** @brief Arguments for selecting the MCU to control a GPIO pin. */
+struct tfm_gpio_service_args_mcu_select {
+	uint32_t pin_number;
+	uint32_t mcu;
```

Comment:
What about

nrf_gpio_pin_mcusel_t mcu;

then users can go to definition of the enum and see possible values.

Reply:
Size of enum can be different depending on compiler flag or compiler used. I assume this needs to be binary compatible.

Reply:
Can document this though.